tests/Crypto/tst_cryptorequests/tst_cryptorequests.cpp

```diff
@@ -386,18 +388,59 @@ void tst_cryptorequests::validateCertificateChain()
 
 void tst_cryptorequests::signVerify()
 {
-    // TODO: sign/verify not yet implemented in test plugin.
-    QByteArray plaintext = "Test plaintext data", signature;
+    // Generate RSA key for signing
+    // ----------------------------
+
+    // Create key template
+    Key keyTemplate;
+    keyTemplate.setSize(1024);
```

Comment:
for RSA keys, the key size is set via the RsaKeyPairGenerationParameters (modulusLength)